.github/workflows/push-changes.yaml

```diff
@@ -0,0 +1,25 @@
+name: Push Changes
+
+on:
+  push:
+    branches: [main]
+  workflow_run:
+    workflows:
+      - Cypress
+      - Docker
+    types:
+      - completed
+
+jobs:
+  dispatch:
+    strategy:
+      matrix:
+        repo: ['devfile/registry-support']
+    runs-on: ubuntu-latest
+    steps:
+      - name: Repository dispatch
+        uses: peter-evans/repository-dispatch@v1
+        with:
+          token: ${{ secrets.REPO_ACCESS_TOKEN }}
```

Comment:
we need to create this token

Reply:
from https://github.com/peter-evans/repository-dispatch#token

> This action creates repository_dispatch events. The default GITHUB_TOKEN does not have scopes to do this so a repo scoped PAT created on a user with write access to the target repository is required. If you will be dispatching to a public repository then you can use the more limited public_repo scope.

Reply:
Also: A user with admin access to the repository needs to add the secret

Reply:
created